### ISSUE TITLE
ACC-253.1 (Level AA) _ Modify Modules _ Navigation menu _ No visible label provided 2

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3133]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Navigation menu
Affected Check Point:	ACC-253.1
Short text:	ACC-253.1 (Level AA) _ Modify Modules _ Navigation menu _ No visible label provided 2

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.

3. Issue Type: Manual
Observed Behavior:
The navigation menu i.e. ‘garden-kyma-stage…’ within the main header shell bar region using which the user can select a particular cluster from the overview of clusters doesn’t have a visible label which describes the purpose of the Ui. 
Screenshot:
 

Expected Behavior:
An appropriate and meaningful visible label must be provided for  navigation menu i.e. ‘garden-kyma-stage…’ within the main header shell bar region which describes the purpose of the Ui.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[Uploading ACC-253.1 (Level AA) _ Modify Modules _ Navigation menu _ No visible label provided 2.docx…]()

**Comment by shubha1090** [comment]
[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659479/Issue.processing-09.2021.002.docx)

**Comment by mrCherry97** [comment]
No response.

**Comment by shubha1090** [comment]
[ACC-253.1 (Level AA) _ Modify Modules _ Navigation menu _ No visible label provided 2.docx](https://github.com/user-attachments/files/19087261/ACC-253.1.Level.AA._.Modify.Modules._.Navigation.menu._.No.visible.label.provided.2.docx)

**Comment by mrCherry97** [comment]
Instead of the visible label, use an invisible `Switch cluster`.
